### PR TITLE
adds update functionality and a form to fill out on the edit page

### PR DESCRIPTION
**Pull request by cmnorrisii**
Also adds testing for the edit page

### CODE REVIEW

File: app/controllers/collections_controller.rb [hunk 1]
```diff
@@ -3,4 +3,28 @@ def index
         collections = Collection.all 
         render json: collections
     end
+
+    def create
```
Comment (Genemichael25) [inline app/controllers/collections_controller.rb:7]: You've got create method that's not a part of the trello card corresponding. Should be removed.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -24,18 +24,34 @@ import mockCollections from "./mockCollections"
       .then((payload)=> setCollections(payload))
       .catch((error)=> console.log(error))
     }
+    
+      const updateCollection = (collection, id) => {
```
Comment (Genemichael25) [inline app/javascript/components/App.js:28]: Needs indentation fixed here

File: app/javascript/components/pages/CollectionEdit.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          />
+        </Routes>
+      </MemoryRouter>
+    )
+  })
+  it("renders an edit page without crashing", () => {
+    const pageTitle = screen.getByText("Edit Cpt. Rex")
+    expect(pageTitle).toBeInTheDocument()
+  })
+  it("has fillable forms for updating an apartment", () => {
```
Comment (Genemichael25) [inline app/javascript/components/pages/CollectionEdit.test.js:25]: apartment?

**Review by Genemichael25 — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/CollectionEdit.test.js (hunk 3 above)
Comment (cmnorrisii) [reply app/javascript/components/pages/CollectionEdit.test.js]: whoops

File: app/javascript/components/pages/CollectionEdit.js [hunk 4]
```diff
@@ -1,9 +1,110 @@
...
+            />
+            <Label for="image">Image</Label>
+          </FormGroup>
+
+          <Button onClick={handleSubmit}>Update {collectionName}</Button>
+        </Form>
+      </div>
+    )
+  } else {
+    return <p>Loading... Please Wait</p>
```
Comment (Genemichael25) [inline app/javascript/components/pages/CollectionEdit.js:107]: They will just wait?

File: app/javascript/components/pages/CollectionEdit.test.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        </Routes>
+      </MemoryRouter>
+    )
+  })
+  it("renders an edit page without crashing", () => {
+    const pageTitle = screen.getByText("Edit Cpt. Rex")
+    expect(pageTitle).toBeInTheDocument()
+  })
+  it("has fillable forms for updating a collection", () => {
+    const formAddress = screen.getByText(/name/i)
```
Comment (Genemichael25) [inline app/javascript/components/pages/CollectionEdit.test.js:26]: You still have some stuff left over from apartment app

**Review by Genemichael25 — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/CollectionEdit.js (hunk 4 above)
Comment (cmnorrisii) [reply app/javascript/components/pages/CollectionEdit.js:107]: its only until editCollection has data to render

File: app/javascript/components/pages/CollectionEdit.test.js [hunk 6]
```diff
@@ -23,19 +23,19 @@ describe("<CollectionEdit />", () => {
     expect(pageTitle).toBeInTheDocument()
   })
   it("has fillable forms for updating a collection", () => {
-    const formAddress = screen.getByText(/name/i)
+    const formName = screen.getByText(/name/i)
     expect(formAddress.getAttribute("for")).toEqual("name")
```
Comment (Genemichael25) [inline app/javascript/components/pages/CollectionEdit.test.js:27]: Is this test passing?

**Review by Genemichael25 — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/CollectionEdit.test.js (hunk 6 above)
Comment (cmnorrisii) [reply app/javascript/components/pages/CollectionEdit.test.js]: .... it is now :(

Comment (cmnorrisii) [reply app/javascript/components/pages/CollectionEdit.test.js]: sorry Gene lol

**Review by Genemichael25 — APPROVED** [review]

**Pull request merged**